Simulation/TSoG_X1_Sim.m

```diff
@@ -92,8 +99,12 @@
     Results.Time(i)  = SimData.Time;
     Results.FSM_state(i)     = SimData.Plane.FSM_state;
     Results.PitchInput(i)    = SimData.TestCase.GetPitch(SimData.Time);
-    Results.ThrottleInput(i) = SimData.TestCase.GetThrottle(SimData.Time);
-
+    Results.ThrottleInput(i) = SimData.battery_ratio*SimData.TestCase.GetThrottle(SimData.Time);
+    
+    #updates on battery for the next iteration
+    Results.battery_status(i) = Results.battery_status(i-1) - SimData.battery_update;
```

Comment:
We should probably be updating the `battery_status/charge` somewhere in derivatives and remove it from `SimData` because it really is the derivative of the battery charge.